src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -137,34 +133,48 @@ export async function promptForArguments( preselectedOptions: InstanceOptions, d
 		elasticsearch: preselectedOptions.elasticsearch || defaultOptions.elasticsearch || DEV_ENVIRONMENT_DEFAULTS.elasticsearchVersion,
 		mariadb: preselectedOptions.mariadb || defaultOptions.mariadb || DEV_ENVIRONMENT_DEFAULTS.mariadbVersion,
 		mediaRedirectDomain: preselectedOptions.mediaRedirectDomain || '',
-		wordpress: {},
-		muPlugins: {},
-		clientCode: {},
+		wordpress: {
+			mode: 'image',
+		},
+		muPlugins: {
+			mode: 'image',
+		},
+		clientCode: {
+			mode: 'image',
+		},
 		statsd: false,
 		phpmyadmin: false,
 		xdebug: false,
+		siteSlug: '',
```

Comment:
Cannot assign object literal to `instanceData` because property `siteSlug` is missing in  object literal [1] but exists in  `InstanceData` [2].